lib/ansible/modules/cloud/amazon/elasticache.py

```diff
@@ -140,19 +142,19 @@
     state: rebooted
 
 """
-
-import sys
-import time
-
+from time import sleep
+# import module snippets
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.ec2 import ec2_argument_spec, get_aws_connection_info, boto3_conn
```

Comment:
You can import HAS_BOTO3 from module_utils.ec2 as well instead of setting it in your try/except below.